pcdsdevices/device_types.py

```diff
@@ -10,3 +10,4 @@
 from .pulsepicker import PulsePicker
 from .slits import Slits
 from .valve import Stopper, GateValve
+from .sequencer import EventSequencer
```

Comment:
Can you sort this into the alphabetized spot? I feel like having a set way to order imports is very useful when there are tons of imports in a module.